tests/end_to_end/end_to_end.cpp

```diff
@@ -853,3 +988,43 @@ TEST_CASE("create_map_name", "[end_to_end]")
         value++;
     }
 }
+
+static void
+_xdp_reflect_packet_test(ebpf_execution_type_t execution_type, ADDRESS_FAMILY address_family = AF_INET)
```

Comment:
```suggestion
_xdp_reflect_packet_test(ebpf_execution_type_t execution_type, ADDRESS_FAMILY address_family)
```
don't default to IPv4